plat/xilinx/zynqmp/pm_service/pm_api_sys.c

```diff
@@ -356,7 +356,10 @@ enum pm_ret_status pm_get_api_version(unsigned int *version)
  */
 enum pm_ret_status pm_set_configuration(unsigned int phys_addr)
 {
-	return PM_RET_ERROR_NOTSUPPORTED;
+	uint32_t payload[PAYLOAD_ARG_CNT];
```

Comment:
Hope the PMU will verify the config and the object received from non-secure world.